### PR TITLE
Update the introduction to step-64.

### PR DESCRIPTION
**Pull request by bangerth**
While there also adjust the dependencies.

### CODE REVIEW

**Comment by bangerth** [comment]
@Rombur @masterleinad @kronbichler: FYI.

**Comment by bangerth** [comment]
In reference to #8012.

File: examples/step-64/doc/builds-on [hunk 1]
```diff
@@ -1 +1 @@
-step-37
+step-6 step-37
```
Comment (masterleinad) [inline examples/step-64/doc/builds-on]: Why "step-6"? Maybe, rather "step-7" (Helmholtz)?

File: examples/step-64/doc/intro.dox [hunk 2]
```diff
@@ -68,38 +79,62 @@ If no memory space is specified, the default is MemorySpace::Host.
...
+@endcode
+The `relevant_rw_vector` is an object that stores a subset of all
+elements of the vector. Typically, these are the
+@ref GlossLocallyRelevantDof "locally relevant DoFs",
+which implies that they overlap between different MPI
+processes. Consequently, the elements stored in that vector on one
+machine may not coincide with the ones stored by the GPU on that
+machine, requiring MPI communication to import them. 
+
+In all of these cases, while importing a vector, values can either by
```
Comment (masterleinad) [inline examples/step-64/doc/intro.dox]:
```suggestion
In all of these cases, while importing a vector, values can either be
```

**Comment by bangerth** [comment]
Comments addressed.

**Pull request merged**